rpm/firejail.spec

```diff
@@ -49,12 +56,21 @@ rm -rf %{buildroot}%{_datadir}/bash-completion
 %files
 %license COPYING
 %defattr(-,root,root,-)
-%attr(4755, -, -) %{_bindir}/firejail
+%attr(4755, -, -) %{_bindir}/%{name}
 %{_bindir}/firecfg
```

Comment:
I think you could move firecfg into the profiles package too. It's for creating /usr/local/bin symlinks for binaries found in the profiles, so it's much use without them. You could also exclude %libdir/firecfg.config, which just lists all the profiles for that tool.

Reply:
Good idea. Thanks!